RNA-seq/00b-additional_approaches.md

```diff
@@ -63,74 +61,33 @@ You can import quantifications from Salmon with `tximeta` and follow a similar p
 
 If you have time to use `tximeta` with your own data, we encourage you to try it out, follow along the package documentation, and let us know what you think!
 
-`tximeta` is not installed on the Docker image we provide you, but it can be installed by running the following command in the R console:
+`tximeta` is not installed on the RStudio Server we use for training, but it can be installed by running the following command in the R console:
```

Comment:
We could install it... should we?

Reply:
That did occur to me, but then I thought it would be a nice way to have folks install an R package themselves if they want to use `tximeta`.